x-pack/plugins/enterprise_search/server/routes/app_search/curations.ts

```diff
@@ -115,4 +115,21 @@ export function registerCurationsRoutes({
       path: '/as/engines/:engineName/curations/find_or_create',
     })
   );
+
+  router.get(
+    {
+      path: '/api/app_search/engines/{engineName}/curation_search',
+      validate: {
+        params: schema.object({
+          engineName: schema.string(),
+        }),
+        query: schema.object({
+          query: schema.string(),
+        }),
+      },
+    },
+    enterpriseSearchRequestHandler.createRequest({
+      path: '/api/as/v1/engines/:engineName/search.json',
```

Comment:
Similarly, I mentioned this a while back in sync but I have no idea what search endpoint we want to be using here. Our options are:

- the public API endpoint (this one)
- the query tester endpoint?
- the relevance tuning endpoint?? (which the standalone UI is using)

I think it mostly just matters for analytics implications 🤷 I left it as the public one for now but def happy to change it if others feel more strongly

Reply:
I think this will be the main issue, is that these searches will be recorded in analytic events. Could you capture a tech debt item to do something about later?

Reply:
For sure! https://github.com/elastic/app-search-team/issues/1631